elements/pl-number-input/pl-number-input.py

```diff
@@ -183,21 +183,51 @@ def parse(element_html, data):
         data['format_errors'][name] = 'No submitted answer.'
         data['submitted_answers'][name] = None
         return
-
-    # Convert to float or complex
-    try:
-        a_sub_parsed = pl.string_to_number(a_sub, allow_complex=allow_complex)
-        if a_sub_parsed is None:
-            raise ValueError('invalid submitted answer (wrong type)')
-        if not np.isfinite(a_sub_parsed):
-            raise ValueError('invalid submitted answer (not finite)')
+        # Accept fractional input, then convert it to a float or decimal
+        # (only works for certain integer addition/subtraction problems)
+    elif '/' in a_sub:
+        a_sub = a_sub.strip()
```

Comment:
is this `strip()` necessary?

Reply:
I just realized .split() makes a separated list, so it is not necessary. Removed .strip() in the code.